hadoop-ozone/client/src/test/java/org/apache/hadoop/ozone/client/checksum/TestReplicatedFileChecksumHelper.java

```diff
@@ -77,9 +80,10 @@
 import static org.mockito.Mockito.mock;
 
 /**
- * Unit tests for ReplicatedFileChecksumHelper class.
+ * Unit tests for Replicated and EC FileChecksumHelper class.
  */
 public class TestReplicatedFileChecksumHelper {
```

Comment:
Just a minor change.
Since the unit test is combining Replicated and EC FileChecksumHelper tests, we can change the test class name to `TestReplicatedAndECFileChecksumHelper`.

Reply:
Let's simplify `TestReplicatedAndECFileChecksumHelper` to `TestFileChecksumHelper`.